### ISSUE TITLE
Stylesheet file is empty

### ISSUE THREAD

**Issue by KTim21** [nextcloud/deck#303]
The stylesheet file "css/deck/57f719b4d9d986f7eb77518be895068e-style.css" is empty. Deck is not usable.

### General server configuration

**Operating system:** Linux aktiv 4.4.0-83-generic #106-Ubuntu SMP Mon Jun 26 17:54:43 UTC 2017 x86_64

**Web server:** Apache/2.4.18 (Ubuntu) (apache2handler)

**Database:** mysql 5.7.19

**PHP version:** 7.0.22-0ubuntu0.16.04.1
<details>
        <summary>PHP-modules loaded</summary>

```
 - Core
 - date
 - libxml
 - openssl
 - pcre
 - zlib
 - filter
 - hash
 - Reflection
 - SPL
 - session
 - standard
 - apache2handler
 - mysqlnd
 - PDO
 - xml
 - calendar
 - ctype
 - curl
 - dom
 - mbstring
 - fileinfo
 - ftp
 - gd
 - gettext
 - iconv
 - igbinary
 - json
 - exif
 - mcrypt
 - mysqli
 - pdo_mysql
 - Phar
 - posix
 - readline
 - redis
 - shmop
 - SimpleXML
 - sockets
 - solr
 - sysvmsg
 - sysvsem
 - sysvshm
 - tokenizer
 - wddx
 - xmlreader
 - xmlwriter
 - xsl
 - zip
 - Zend OPcache
```
</details>

### Nextcloud configuration

**Nextcloud version:** 12.0.2 - 12.0.2.0

**Updated from an older Nextcloud/ownCloud or fresh install:** updated

**Where did you install Nextcloud from:**  YOUR ANSWER HERE

**Are you using external storage, if yes which one:** files_external is disabled

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

<details>
        <summary>Signing status</summary>

```
[]
```
</details>

<details>
        <summary>Enabled apps</summary>

```
 - activity: 2.5.2
 - admin_audit: 1.2.0
 - bruteforcesettings: 1.0.2
 - calendar: 1.5.4
 - comments: 1.2.0
 - dav: 1.3.0
 - deck: 0.2.2
 - federatedfilesharing: 1.2.0
 - federation: 1.2.0
 - files: 1.7.2
 - files_pdfviewer: 1.1.1
 - files_sharing: 1.4.0
 - files_texteditor: 2.4.1
 - files_trashbin: 1.2.0
 - files_versions: 1.5.0
 - files_videoplayer: 1.1.0
 - firstrunwizard: 2.1
 - gallery: 17.0.0
 - groupfolders: 1.1.0
 - impersonate: 1.0.1
 - issuetemplate: 0.2.2
 - keeweb: 0.4.0
 - logreader: 2.0.0
 - lookup_server_connector: 1.0.0
 - metadata: 0.5.0
 - nextant: 1.0.8
 - notifications: 2.0.0
 - oauth2: 1.0.5
 - password_policy: 1.2.2
 - piwik: 0.3.1
 - polls: 0.7.3
 - previewgenerator: 1.0.6
 - provisioning_api: 1.2.0
 - ransomware_protection: 1.0.3
 - registration: 0.3.0
 - richdocuments: 1.12.34
 - serverinfo: 1.2.0
 - sharebymail: 1.2.0
 - spreed: 2.0.1
 - twofactor_backupcodes: 1.1.1
 - updatenotification: 1.2.0
 - workflowengine: 1.2.0
```
</details>

<details>
        <summary>Disabled apps</summary>

```
 - encryption
 - files_external
 - nextcloud_announcements
 - ojsxc
 - ownpad
 - survey_client
 - systemtags
 - theming
 - user_external
 - user_ldap
```
</details>

<details>
        <summary>Content of config/config.php</summary>

```
{
    "instanceid": "ocej0rldisde",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "***REMOVED SENSITIVE VALUE***"
    ],
    "datadirectory": "\/var\/www\/html\/nextcloud\/data",
    "overwrite.cli.url": "***REMOVED SENSITIVE VALUE***e",
    "dbtype": "mysql",
    "version": "12.0.2.0",
    "dbname": "nxtcloud",
    "dbhost": "localhost",
    "dbtableprefix": "oc_",
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "logtimezone": "UTC",
    "installed": true,
    "maintenance": false,
    "loglevel": 3,
    "skeletondirectory": "",
    "mail_from_address": "noreply",
    "mail_smtpmode": "smtp",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "enabledPreviewProviders": [
        "OC\\Preview\\PNG",
        "OC\\Preview\\JPEG",
        "OC\\Preview\\GIF",
        "OC\\Preview\\BMP",
        "OC\\Preview\\XBitmap",
        "OC\\Preview\\MP3",
        "OC\\Preview\\TXT",
        "OC\\Preview\\MarkDown",
        "OC\\Preview\\TIFF"
    ],
    "mail_smtpsecure": "tls",
    "mail_smtpauthtype": "LOGIN",
    "mail_smtpport": "25",
    "mail_smtpauth": 1,
    "mail_smtphost": "smtp.1und1.de",
    "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
    "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
    "appstore.experimental.enabled": true,
    "knowledgebaseenabled": false,
    "theme": "***REMOVED SENSITIVE VALUE***",
    "memcache.local": "\\OC\\Memcache\\Redis",
    "memcache.locking": "\\OC\\Memcache\\Redis",
    "redis": {
        "host": "localhost",
        "port": 6379
    },
    "htaccess.RewriteBase": "\/",
    "default_language": "de",
    "updater.release.channel": "production"
}
```
</details>

### Client configuration

**Browser:** Mozilla/5.0 (X11; Linux x86_64; rv:52.0) Gecko/20100101 Firefox/52.0

**Operating system:** YOUR ANSWER HERE

### Logs

<details>
        <summary>Web server error log</summary>

```
[Fri Sep 15 06:25:05.856257 2017] [mpm_prefork:notice] [pid 20734] AH00163: Apache/2.4.18 (Ubuntu) OpenSSL/1.0.2g configured -- resuming normal operations
[Fri Sep 15 06:25:05.856319 2017] [core:notice] [pid 20734] AH00094: Command line: '/usr/sbin/apache2'
[Fri Sep 15 17:05:44.321727 2017] [authz_core:error] [pid 28814] [client 46.183.103.8:11414] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Sep 15 17:06:01.481159 2017] [authz_core:error] [pid 28917] [client 46.183.103.8:6444] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Sep 15 17:06:36.945652 2017] [authz_core:error] [pid 28724] [client 46.183.103.8:31455] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Sep 15 17:06:46.603180 2017] [authz_core:error] [pid 28487] [client 46.183.103.8:28108] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Sep 15 17:06:55.393066 2017] [authz_core:error] [pid 28928] [client 46.183.103.8:26929] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Sep 15 17:07:10.843302 2017] [authz_core:error] [pid 28915] [client 46.183.103.8:10583] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Sep 15 17:07:35.885293 2017] [authz_core:error] [pid 28724] [client 46.183.103.8:29133] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Sep 15 17:11:13.786485 2017] [:error] [pid 28809] [client 78.51.97.25:61432] PHP Fatal error:  Uncaught RedisException: Redis server went away in /var/www/html/nextcloud/lib/private/Memcache/Redis.php:52$
[Fri Sep 15 17:11:13.786684 2017] [:error] [pid 28809] [client 78.51.97.25:61432] PHP Fatal error:  Uncaught RedisException: Redis server went away in /var/www/html/nextcloud/lib/private/Memcache/Redis.php:52$
[Fri Sep 15 17:27:01.321756 2017] [authz_core:error] [pid 29288] [client 46.183.103.8:12294] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Sep 15 17:27:05.104207 2017] [authz_core:error] [pid 29327] [client 46.183.103.8:26122] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
```
</details>

<details>
        <summary>Nextcloud log (data/nextcloud.log)</summary>
</details>

<details>
        <summary>Browser log</summary>

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>

**Comment by juliushaertl** [comment]
Looks like some recent issues with scss generation in server. Can you check if the issue still exists with NC 12.0.3? The RC is already out for testing.

**Comment by KTim21** [comment]
yes, fixed with 12.0.3
